opentracing_shim/src/test/java/io/opentelemetry/opentracingshim/testbed/TestUtils.java

```diff
@@ -67,7 +67,7 @@ public Integer call() throws Exception {
         new Condition() {
           @Override
           public boolean check(Span span) {
-            AttributeValue attrValue = span.getAttributes().getAttributeMap().get(key);
+            AttributeValue attrValue = span.getAttributes().getAttributeMapMap().get(key);
```

Comment:
did bumping the autovalue version make this method change?

Reply:
No, the `options.compilerArgs += ["-Xlint:-deprecation"]` I probably should do that separately.

Reply:
Done in a different PR.